arcgis-ios-sdk-samples/Scenes/Feature layer extrusion/README.md

```diff
@@ -1,21 +1,34 @@
 # Feature layer extrusion
 
-This sample demonstrates how to apply extrusion to a renderer on a feature layer.
+Extrude features based on their attributes.
 
-![](image1.png)
+![Feature layer extrusion sample](feature-layer-extrusion.png)
+
+## Use case
+
+Extrusion is the process of stretching a flat, 2D shape vertically to create a 3D object in a scene. For example, you can extrude building polygons by a height value to create three-dimensional building shapes.
+
+## How to use the sample
+
+Tap the bottom buttons to switch between using population density and total population for extrusion. Higher extrusion directly corresponds to higher attribute values.
 
 ## How it works
-To extrude features from an `AGSFeatureLayer`:
-
-1. Create an `AGSServiceFeatureTable` from an URL.
-2. Create a feature layer from service feature table.
-Make sure to set rendering mode to dynamic, `layer.renderingMode = .dynamic`.
-3. Apply a SimpleRenderer to the feature layer.
-4. Set `ExtrusionMode` of render, `renderer.sceneProperties.extrusionMode = .baseHeight`.
-5. Set extrusion expression of renderer, `renderer.sceneProperties.extrusionExpression = "[POP2007]/ 10"`.
-
-## Features
-- FeatureLayer
-- SceneProperties
-- ServiceFeatureTable
-- FeatureLayer
+
+1. Create an `AGSServiceFeatureTable` from a URL.
+2. Create an `AGSFeatureLayer` from the service feature table.
+    * Make sure to set the `renderingMode` property to `dynamic`.
+3. Apply an `AGSSimpleRenderer` to the feature layer.
+4. Set the renderer's `extrusionMode` property to `absoluteHeight`.
+5. Set the renderer's `extrusionExpression` property to `"[POP2007]/ 10"`.
```

Comment:
In the code the expression is "[POP2007]/ 10" rather than "[POP2007] / 10", so I left this unchanged.